index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main id="main">
+      <div class="title-container">
+        <h1 id="title">Yemi Alade</h1>
+        <div id="subtitle">The Queen of Afro Pop</div>
+      </div>
+
+      <div class="img-div">
+        <img
+          id="image"
+          src="https://i.postimg.cc/JhDzVP9p/yemi-alade.jpg"
+          alt="Yemi Alade"
+        />
+        <div id="img-caption">
+          <i> Yemi Alade posing in a photoshoot</i>
+        </div>
+      </div>
+
+      <div id="tribute-info">
+        <h2>Biography</h2>
+        <section>
+          <p>
+            Yemi Alade is a Nigerian Afropop singer, songwriter, actress and
+            activist. She won the Peak Talent Show in 2009 after which she
+            signed to Effyzzie Music Group, and had a hit with her single
+            "Johnny" in 2014. She won the MTV African Music Awards for Best
+            female in 2015 and 2016 consecutively and was nominated for Artiste
+            of the Year in 2015, making her to first female to win the MAMAs for
+            Best female consecutively twice and nominated for Artiste of the
+            Year. In 2015, Alade was the first Nigerian female to be nominated
+            for the MTV Europe Music Awards (EMAs) for Best African Act. Yemi is
+            also applauded for her creative and overwhelming stage performances,
+            fashion and music videos, She has won The Headies Award for Best
+            Performer twice, 2018 and 2019. The slot was initiated at The
+            Headies 2018 and she was the first artiste to win the category.
+            Alade sings in English, Igbo, Pidgin, Yoruba, French, Swahili n
+            Portuguese
+          </p>
+        </section>
+
+        <section></section>
```

Comment:
## OPTIONAL

- [ ] Please phrase the comments politely, as suggestions instead of orders.
- [ ] This is a polite suggestion: **This two section tags appear to be unused code, I think it should be cleared in order to make the code readable**

Reply:
his section tag appears unused. I think it should be cleared in order to make the code more readable